styles/styles.css

```diff
@@ -526,6 +541,10 @@ main .section.gray-gradient-background[data-background-image] {
   :root {
     --nav-height: 75px;
   }
+
+  main .section.angled-inverted-background.hero-container .hero {
+    clip-path: polygon(0 0, 100% 0, 100% calc(100% - 10vw), 0 100%);
```

Comment:
Breakpoints translated from `10vw` at `< 768px` and `5vw` `>= 768px` to be `10vw` below `900px` breakpoint and `5vw` at `900px` breakpoint and above.